File: BOJ/CLASS 2+/4153.js
// 직각삼각형

const input = require('fs')
  .readFileSync('예제.txt')
  .toString()
  .trim()
  .split('\n');

function solution(arr) {
  let answer = [];

  for (let i = 0; i < arr.length - 1; i++) {
    const [x, y, z] = arr[i]
      .split(' ')
      .sort((a, b) => a - b)
      .map(Number);

    if (x ** 2 + y ** 2 === z ** 2) answer.push('right');
    else answer.push('wrong');
  }

  return answer.join('\n');
}

console.log(solution(input));

File: BOJ/CLASS 2/Python/4153.py
# 직각삼각형
while(True):
    # a, b, c = sorted(list(map(int, input().split())))
    a, b, c = sorted(map(int, input().split()))

    if a == 0 and b == 0 and c == 0: break

    print('right' if (a ** 2 + b ** 2) == c ** 2 else 'wrong')

"""
sort() 메서드 사용: 리스트를 제자리에서(in-place) 정렬, None 반환
sorted() 함수 사용: 정렬된 새로운 리스트를 반환
"""